services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistorySpec.groovy

```diff
@@ -114,24 +153,30 @@ class FlowHistorySpec extends HealthCheckSpecification {
         flowHistory.isEmpty()
     }
 
-    void checkHistoryCreateAction(flowHistory, flowId) {
-        assert flowHistory.flowId == flowId
-        assert flowHistory.action == createAction
-        assert flowHistory.taskId
-        assert flowHistory.histories.action[0] == createHistoryAction
-    }
+    void checkHistoryAction(flowHistory, flowId, action) {
+        String actionMessage
+        String historyMessage
+        if (action == "createV1"){
+            actionMessage = createAction
+            historyMessage = createHistoryActionV1
+        } else if( action == "createV2") {
+            actionMessage = createAction
+            historyMessage = createHistoryActionV2
+        } else if ( action == "update") {
+            actionMessage = updateAction
+            historyMessage = updateHistoryAction
+        }
```

Comment:
Bad design. 
1. avoid 'magic values'
2. do not try to combine 3 different methods in one method with 3 ifs. create 3 separate methods instead.